roles/glance/templates/etc/glance/policy.json

```diff
@@ -16,8 +16,13 @@
     "upload_image": "",
 
     "delete_image_location": "",
+{% if allow_image_locations|default('False')|bool %}
     "get_image_location": "",
     "set_image_location": "",
+{% else %}
+    "get_image_location": "role:admin or role:cloud_admin",
```

Comment:
I think I see the logic here.  The default case of allow_image_locations is False will follow the else so it will get locked down to admins.
If allow_image_locations were True then it will emit w/o the admin lockdown, thus any user could manage.